CMakeLists.txt

```diff
@@ -144,7 +144,7 @@ set(POSITION_INDEPENDENT_CODE True)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 set(CMAKE_CXX_FLAGS
-    "${CMAKE_CXX_FLAGS} -mavx2 -mfma -mavx -mf16c -mbmi2 -march=native -D USE_VELOX_COMMON_BASE"
+    "${CMAKE_CXX_FLAGS} -mavx2 -mfma -mavx -mf16c -mbmi2 -march=native -mno-avx512f -D USE_VELOX_COMMON_BASE"
```

Comment:
I see `-march=native` is still there. Does this not need to change? Would xSIMD somehow depress the error on M1?
```
clang: error: the clang compiler does not support '-march=native'
```